axopy/pipeline/common.py

```diff
@@ -404,15 +404,40 @@ class Estimator(Block):
     estimator : object
         An object implementing the scikit-learn Estimator interface (i.e.
         implementing ``fit`` and ``predict`` methods).
+    return_proba : boolean, optional (default: False)
+        If True, return probability estimates.
+    return_log_proba : boolean, optional (default: False)
+        If True, return log of probability estimates.
     """
 
-    def __init__(self, estimator):
+    def __init__(self, estimator, return_proba=False):
         super(Estimator, self).__init__()
         self.estimator = estimator
+        self.return_proba = return_proba
+        self.return_log_proba = return_log_proba
+        self._check_estimator()
```

Comment:
Could you also add a check for the condition that both `return_proba=True` and `return_log_proba=True`? I'm leaning toward just issuing a warning saying `process` will default to `predict_proba`. I don't have a strong preference for that over an exception though.